### ISSUE TITLE
 Why is the result so unnatural?

### ISSUE THREAD

**Issue by facelabmania** [iperov/DeepFaceLab#900]
![나에게](https://user-images.githubusercontent.com/71271165/93523141-8bc64600-f96d-11ea-9bd3-bef00173cae0.png)

saehd train 10k....

1650super

0] Which GPU indexes to choose? :
0

[0] Autobackup every N hour ( 0..24 ?:help ) :
0
[n] Write preview history ( y/n ?:help ) : y
[n] Choose image for the preview history ( y/n ) : y
[0] Target iteration :
0
[y] Flip faces randomly ( y/n ?:help ) :
y
[8] Batch_size ( ?:help ) : 4
4
[128] Resolution ( 64-640 ?:help ) :
128
[f] Face type ( h/mf/f/wf/head ?:help ) :
f
[df] AE architecture ( ?:help ) :
df
[256] AutoEncoder dimensions ( 32-1024 ?:help ) :
256
[64] Encoder dimensions ( 16-256 ?:help ) :
64
[64] Decoder dimensions ( 16-256 ?:help ) :
64
[22] Decoder mask dimensions ( 16-256 ?:help ) :
22
[n] Eyes priority ( y/n ?:help ) :
n
[n] Uniform yaw distribution of samples ( y/n ?:help ) :
n
[y] Place models and optimizer on GPU ( y/n ?:help ) :
y
[n] Use learning rate dropout ( n/y/cpu ?:help ) :
n
[y] Enable random warp of samples ( y/n ?:help ) : n
[0.0] GAN power ( 0.0 .. 10.0 ?:help ) :
0.0
[0.0] 'True face' power. ( 0.0000 .. 1.0 ?:help ) :
0.0
[0.0] Face style power ( 0.0..100.0 ?:help ) :
0.0
[0.0] Background style power ( 0.0..100.0 ?:help ) :
0.0
[none] Color transfer for src faceset ( none/rct/lct/mkl/idt/sot ?:help ) :
none
[n] Enable gradient clipping ( y/n ?:help ) :
n
[n] Enable pretraining mode ( y/n ?:help ) :
n

**Comment by test1230-lab** [comment]
only 10k iters?

**Comment by facelabmania** [comment]
> y 10k iters?

no... this is 100k    

mistake wirrten

**Comment by ghost** [comment]
Issue solved / already answered (or it seems like user error), please close it.
